File: Ex6/jmmakiik/Problem1/writing.cxx
#include <iostream>

#include "TTree.h"
#include "TBranch.h"
#include "TFile.h"
#include "TRandom.h"

using namespace std;

int main(){

  Int_t N = 1000;
  Float_t variable[0];
  gRandom->SetSeed(7);
  
  TFile f("writing.root","RECREATE");
  TTree tree1("MyTree","tree");
  tree1.Branch("Random_Gaussian_numbers",&variable,"variable/F");

  for(Int_t i = 0; i < N;i++){
    variable[0] = gRandom->Gaus();
    tree1.Fill();
  }
  
  tree1.Write();
  f.Close();
  
}

File: Ex6/jmmakiik/Problem2/writing.py
import ROOT
from array import array

N = 1000
variable = array("f",[0])
ROOT.gRandom.SetSeed(7)
	
writing = ROOT.TFile("writing.root","RECREATE")
tree1 = ROOT.TTree("MyTree","tree")
tree1.Branch("Random_Gaussian_numbers",variable,"variable/F")
	
for i in range(0,N,1):
    variable[0] = ROOT.gRandom.Gaus()
    tree1.Fill()
	
tree1.Write()
writing.Close()
